### PR TITLE
docs: added commit guideline

### PR DESCRIPTION
**Pull request by kirrg001**
Let me know if you agree.

### CODE REVIEW

**Review by athirasomanath — APPROVED** [review]

**Review by basti1302 — APPROVED** [review]

**Review by aryamohanan — APPROVED** [review]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -63,6 +63,16 @@ Please print, fill out, and sign the [contributor license agreement](https://git
 
 Thank you for your interest in the Instana Node.js project!
 
+## Commit messages
+
+We are adhering to the [Conventional Commits standard](https://www.conventionalcommits.org/en/v1.0.0/), which defines specific commit message types. These types include: chore, docs, feat, fix, refactor, style, or test.
```
[the remark this answers is not in the document]
Comment (kirrg001) [reply CONTRIBUTING.md:68]: Updated the list again 👍 
I think we should use build for deps and chore for any other internal changes.
But it's also fine to use chore for build updates. I think the rule shouldn't be that strict.

[the remark this answers is not in the document]
Comment (kirrg001) [reply CONTRIBUTING.md:68]: The differentiation between user and non user facing changes is more import tbh.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -63,6 +63,16 @@ Please print, fill out, and sign the [contributor license agreement](https://git
...
+
+For any change that is customer-facing and should appear in the changelog file, we require the use of either 'fix' or 'feat' in the commit message. If the change is not customer-facing and is more related to internal tasks or housekeeping, it should use 'chore,' for instance.
+
+When composing commit messages, it's important to use past tense. Additionally, if there is a corresponding ticket or issue associated with the change, please reference it in the commit message.
```
[the remark this answers is not in the document]
Comment (kirrg001) [reply CONTRIBUTING.md:72]: The example below contains the refs.
https://github.com/instana/nodejs/commit/bd3e7554fe21188c3ad10d442e4d72546d5c2267

Not enough? 🤔

**Pull request merged**